src/main/kotlin/Command/commands/ChooseNoodlesCommand.kt

```diff
@@ -0,0 +1,19 @@
+package Command.commands
+
+import Command.base.Cart
+import Command.base.Command
+
+class ChooseNoodlesCommand(val cart : Cart) : Command {
```

Comment:
In theory, the Command only wraps an action of the receiver, isn't it? It shouldn't have any "ui behaviour". Would you do different?